indico/modules/events/papers/templates/management/assignment_list_filter.html

```diff
@@ -0,0 +1,84 @@
+{% from 'forms/_form.html' import form_header, form_footer, form_rows %}
+{% from '_switch.html' import switch %}
+
+{% macro _render_column_selector(item_id, item, filter_choices, is_static_item=false) %}
+    {% set filters = filters['items'] if is_static_item else filters['fields'] %}
+    <div class="label-group list-column">
+        <div class="i-label title-wrapper" data-id="{{ item_id }}">
+            <div class="title">{{ item.title }}</div>
+            <div class="actions">
+                <span class="visibility icon-eye"></span>
+            </div>
+        </div>
+        {% if filter_choices %}
+            <div class="i-label filter">
+                <a class="icon-filter" data-toggle="dropdown"></a>
+                <ul class="dropdown" style="display: none">
+                    {% for value, caption in filter_choices.iteritems() %}
+                        <li>
+                            <input type="checkbox" name="field_{{ item_id }}" value="{{ value }}"
+                                   id="field_{{ item_id }}_option_{{ value }}"
+                                   {% if value|string in filters[item_id]|string %}
```

Comment:
The problem is when the value is `None` (in cases like "Not yet submitted", "No track", etc.), then the check is `if "None" in [None, ...]` which is false and must be true.